crates/std_detect/src/detect/os/x86.rs

```diff
@@ -200,6 +206,7 @@ pub(crate) fn detect_features() -> cache::Initializer {
                     // For AVX-512 the OS also needs to support saving/restoring
                     // the extended state, only then we enable AVX-512 support:
                     if os_avx512_support {
+                        enable(extended_features_eax, 5, Feature::avx512bf16);
```

Comment:
According to the reference, we should get this bit in the output `eax` register of `__cpuid_count(7, 1)`.

Reply:
fixed in new commit